### PR TITLE
notify-api-533 add flask command to promote user to platform admin

### PR DESCRIPTION
**Pull request by terrazoon**

### CODE REVIEW

File: app/dao/__init__.py [hunk 1]
```diff
@@ -17,13 +17,6 @@ def create_instance(self, inst, _commit=True):
         if _commit:
             db.session.commit()
 
-    def update_instance(self, inst, update_dict, _commit=True):
-        # Make sure the id is not included in the update_dict
-        update_dict.pop("id")
-        self.Meta.model.query.filter_by(id=inst.id).update(update_dict)
-        if _commit:
-            db.session.commit()
-
```
Comment (ccostino) [inline app/dao/__init__.py:26]: I'm not sure I follow why this method was removed; wouldn't this still be needed?

**Review by ccostino — COMMENTED** [review]
Thanks, @terrazoon!

This looks great to me, but I have one question about the method that was removed from the DAO model because that seemed out of place to me.

**Comment by terrazoon** [comment]
> Thanks, @terrazoon!
> 
> This looks great to me, but I have one question about the method that was removed from the DAO model because that seemed out of place to me.

Even though I wrote a test for my new method that tested everything, code coverage fell to 94%.   I looked around for a test to write and it seemed like it would be easy to write one for DAOClass.update_instance().  It was not, and in figuring out why I noticed that update_instance() is not actually used anywhere.  By deleting it, I got code coverage to go back to 95%.

File: app/dao/__init__.py (hunk 1 above)
Comment (terrazoon) [reply app/dao/__init__.py:26]: It's never used anywhere.

Comment (ccostino) [reply app/dao/__init__.py:26]: Sure enough it's not!

**Review by ccostino — APPROVED** [review]
Thanks again, @terrazoon!

**Pull request merged**